openssl-dynamic/src/main/c/tcn.h

```diff
@@ -162,34 +142,6 @@ char           *tcn_pstrdup(JNIEnv *, jstring, apr_pool_t *);
         }                                           \
     TCN_END_MACRO
 
-#define TCN_MAX_METHODS 8
-
-#define TCN_MIN(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
I'm actually using this in a pending PR ... I can just add it back unless there is a better alternative to use.